airflow/operators/s3_to_mysql.py

```diff
@@ -0,0 +1,65 @@
+from airflow.hooks.mysql_hook import MySqlHook
+from airflow.models import BaseOperator
+from airflow.providers.aws.hooks.s3 import S3Hook
+from airflow.utils.decorators import apply_defaults
+
+
+class S3ToMySqlTransfer(BaseOperator):
+    """
+    Loads a file from S3 into a MySQL table.
```

Comment:
I think you need to specify it's CSV file.
Other operators supports json as well - this one does not

Reply:
Good point. Actually at first I had "csv" in there. But I wanted to make this operator handle all kind of files. So you can also transfer txt files instead of csv files with it.

Reply:
I could do some validation on the file format if you think that is really needed otherwise you probably will see a mysql error for uncompatible files.

Reply:
You can follow how it works in MySqlToGoogleCloudStorageOperator https://github.com/apache/airflow/blob/master/airflow/operators/mysql_to_gcs.py#L36
It support both Json and csv
Personally I really like the idea of BaseSQLToGoogleCloudStorageOperator as I believe it's the same case here but that's your call.

Reply:
I think that's a bit different. I want to transfer from storage to database.